packages/terra-status-view/src/StatusView.jsx

```diff
@@ -98,29 +98,43 @@ class StatusView extends React.Component {
    * with 40% of the remaining space at the top with 60% on the bottom.
    */
   showAndCenterItems() {
-    const viewHeight = this.innerNode.offsetHeight;
+    const viewHeight = this.innerNode.getBoundingClientRect().height;
+    let newViewHeight = viewHeight;
```

Comment:
do we need to reasign `viewHeight` to a new variable?

Reply:
Yes we need actual viewheight to calculate the padding of the inner view.